check/mypy

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Runs mypy on the repository using a preconfigured mypy.ini file.
+#
+# Usage:
+#     check/mypy [--flags]
+################################################################################
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+echo -e -n "\033[31m"
+mypy --config-file=dev_tools/conf/mypy.ini $@ .
```

Comment:
nit: it's safer with quotes: `"$@"` (which is equivalent to quoting each array element separately)

Reply:
Done